jbcli/jbcli/cli/jb.py

```diff
@@ -563,6 +487,7 @@ def stop(ctx, clean, env):
 def yo_upgrade():
     """ Attempt to upgrade yo juicebox to the current version
     """
+    return
```

Comment:
was this intentional? do we just want to remove this command entirely?

Reply:
i don't specifically remember doing anything with this but chris might have when experimenting with a few changes @chrisgemignani ?

Reply:
Yeah; I think we want to remove this command entirely. I think I was just trying something out. Let's remove entirely @obscure-shadow